contents/docs/libraries/js/index.mdx

```diff
@@ -468,8 +468,8 @@ Since version 1.187.0, when initializing the SDK, you can provide a `before_send
 
 #### Redact URLs in event properties
 
-```
-posthog.init('my_token', {
+```ts
+posthog.init('<ph_project_api_key>', {
```

Comment:
Unrelated: I've always thought developer products should use a key on their examples that is used as a sentinel value in the library code, that would warn customers they need to replace the value with something more useful versus dying silently. Do we have anything similar to this already?

Reply:
Yeah, if you open this page when logged in it should replace it with your real token, but please try it :)